client/components/spinner/index.jsx

```diff
@@ -1,5 +1,12 @@
 /** @format */
 
+/**
+ * If making changes in this file, please also open a PR in
+ * the Automattic/notifications-panel repo, which uses this
```

Comment:
Let's provide the full link

Reply:
Minor point but we also we use `//` for anything that's not jsdoc/dependency/format headers https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/javascript.md#comments